tested/judge/core.py

```diff
@@ -116,6 +118,16 @@ def judge(bundle: Bundle):
     # Do the set-up for the judgement.
     collector = OutputManager(bundle.out)
     collector.add(StartJudgement())
+    if bundle.translations_missing_key:
```

Comment:
I don't really like the fact that this change is needed here.
But it is a useful message and don't know how to do this any better.

Maybe to minimize the change here do something like
```
if bundle.preprocessor_messages:
     collector.add_messages(bundle.preprocessor_messages)
```
     
with those messages being build in translate_parser.py